gluetool/log.py

```diff
@@ -91,6 +91,8 @@ def __init__(self, some_logger):
 # e.g. raw output of commands or API responses.
 VERBOSE = 5
 
+# Our custom "important" loglevel
+IMPORTANT = 21
```

Comment:
I wonder, why did we choose magic number 21 here?

Reply:
See `logging._levelNames` - `IMPORTANT` fits between `INFO` and `WARNING`. Although, I'd recommend making it a slightly higher, e.g. `25`, to make room for any possible additions in the future, between `INFO` and `IMPORTANT` :)